### DISCUSSION TITLE
Crypto-gated Calendly

### DISCUSSION THREAD

**Discussion by paarugsethi** [solana-developers/awesome-blinks#4] [Ideas for Blinks]
- No-code platform for someone to create a blink to pay USDC for a 15/30/60 minute call
- User tweets the blink on Twitter/X
- As soon as someone pays, auto-create a meeting based on the user's calendar availability

**Comment by WaggMe** [comment]
Already working on a blink pattern that solves this! 🔥

**Reply by paarugsethi** [reply]
legend!

**Comment by adlonymous** [comment]
Doesn't Sollinked already provide this functionality + a gated blog? If it's OS, then can just fork it and build a blink on top of it
